packages/data-broker/src/__tests__/services/dhis/pullers/EventsPuller.test.js

```diff
@@ -21,14 +18,6 @@ describe('EventsPuller', () => {
     eventsPuller = new EventsPuller(models, translator);
     dhisApi = createMockDhisApi();
     stubGetDhisApi(dhisApi);
-    buildEventsFromDhisEventAnalytics.mockReturnValue('X');
-  });
-
-  it('pulls', async () => {
-    const result = await eventsPuller.pull([dhisApi], [DATA_GROUPS.POP01_GROUP], {
-      dataElementCodes: ['POP_01'],
-    });
-    expect(result).toEqual(['X']); // not sure why but mocked return value is wrapped into an array
```

Comment:
Decided to scrap this test, as the tests below actually test pulling data with real results, so seems a bit redundant

Reply:
I think the idea for this was a happy path test which is a handy way of quickly understanding a class by looking at the first test, but happy to consolidate too. We won't really need it once we have types.